packages/synapse-react-client/src/components/TableColumnSchemaEditor/TableColumnSchemaEditor.tsx

```diff
@@ -81,32 +58,65 @@ export default function TableColumnSchemaEditor(
       },
     )
 
-  const { mutate, isLoading: isMutating } = useUpdateTable()
+  const { mutate, isLoading: isMutating, error } = useUpdateTableColumns()
 
-  const isLoading = isLoadingEntity || isLoadingColumnModels || isMutating
```

Comment:
Inappropriate to show loading screen when `isMutating`. We should disable the form, not hide it!